arex-storage-web-api/src/main/java/com/arextest/storage/metric/AgentWorkingMetricService.java

```diff
@@ -132,5 +119,37 @@ private void invalidReplayCase(InvalidCaseRequest request) {
       metricListener.recordMatchingCount(METRIC_NAME, tags);
     }
   }
+  private class InvalidTask implements Runnable {
```

Comment:
business prcessing code should not be in `MetricClass`,  the purpose of this class is just metric